apps/ledger-live-mobile/src/screens/AccountSettings/__tests__/EditAccountName.test.tsx

```diff
@@ -0,0 +1,147 @@
+import { render } from "@tests/test-renderer";
+import React from "react";
+import EditAccountName from "../EditAccountName";
+import { ScreenName } from "~/const";
+import { AccountSettingsNavigatorParamList } from "~/components/RootNavigator/types/AccountSettingsNavigator";
+import { RouteProp } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
+import { BaseNavigatorStackParamList } from "~/components/RootNavigator/types/BaseNavigator";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/currencies/index";
+import { genAccount } from "@ledgerhq/live-common/mock/account";
+
+jest.mock("react-redux", () => {
+  const actual = jest.requireActual("react-redux");
+  return {
+    ...actual,
+    useDispatch: () => mockDispatch,
+  };
+});
+
+const mockDispatch = jest.fn();
```

Comment:
Is there no way to not mock the dispatch function and just check the effect of the component on the Redux store ?

Reply:
Currently not, the store is available only for renderHook,

I can try to implement it

Reply:
Done